distribution/lib/Standard/Table/0.0.0-dev/src/Io/Excel.enso

```diff
@@ -0,0 +1,184 @@
+from Standard.Base import Integer, Text, Nothing, Boolean, Illegal_Argument_Error, Any, Error, Panic, File, Vector, False, IO
+from Standard.Base.Error.Problem_Behavior as Problem_Behavior_Module import Problem_Behavior
+
+import Standard.Table.Data.Table
+from Standard.Table.Error as Error_Module import Invalid_Location
+
+polyglot java import org.enso.table.format.xlsx.Range as Java_Range
+polyglot java import org.enso.table.format.xlsx.Reader
+
+polyglot java import java.lang.IllegalArgumentException
+polyglot java import java.io.IOException
+polyglot java import org.apache.poi.UnsupportedFileFormatException
+
+## Specified the part of an Excel Workbook to Read
+type Excel_Section
+    ## Gets a list of sheets within a workbook
+    type Sheet_Names
+
+    ## Gets a list of named ranges within a workbook
+    type Range_Names
+
+    ## Gets the data from a specific sheet. Column names are the Excel column names.
+    type Sheet (sheet:(Integer|Text)) (skip_rows:(Integer|Nothing)=Nothing) (row_limit:(Integer|Nothing)=Nothing)
+
+    ## Gets a specific range (taking either a defined name or external style address) from the workbook
+    type Range (address:(Text|Excel_Range)) (skip_rows:(Integer|Nothing)=Nothing) (row_limit:(Integer|Nothing)=Nothing)
+
+
+## Specifies a range within Excel and provides various ways of
+type Excel_Range
+    type Excel_Range java_range:Java_Range
+
+    ## Gets the name of the sheet
```

Comment:
```suggestion
    ## Gets the name of the sheet.
```

I think we should end sentences in docs with a full stop (applies to all other docs too).

Reply:
Also I noticed that this class has docs in two formats: imperative ("Get something") and descriptive ("Gets something"). Ideally we should have a style guide and be consistent across whole codebase, but at least would be good to be consistent across the single module.